nfc/license.txt

```diff
@@ -2,4 +2,8 @@
  * Copyright (c) 2018 Nordic Semiconductor ASA
  *
  * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ *
+ * The following tag tells the "west ncs-sbom" tool which files are covered
```

Comment:
Should we mention the Telit here @grochu ?
I am not sure how we should handle a licence here.

Reply:
Good point @KAGA164, I think the copyright in _nfc/license.txt_ is not correct as it should reproduce the Telit's copyright (to satisfy the conditions listed in the license in the headers: https://github.com/nrfconnect/sdk-nrfxlib/blob/main/nfc/include/nfc_t2t_lib.h and https://github.com/nrfconnect/sdk-nrfxlib/blob/main/nfc/include/nfc_t4t_lib.h). I am not sure though if Nordic should also be mentioned in the copyright as the library also contains Nordic code additions/modifications. When discussing this in nRF5 SDK context I think the conclusion was that just the original authors of the source code (Telit) should be listed in the copyright. Maybe @jdstapes has an answer?

Reply:
For now, the `west ncs-sbom` script does not report copyright information, so updating `Copyright (c)` line is not needed for this PR. For now we are interested in license identifier which is correct. If the license identifier and the list of files that it covers is correct in this PR, then please accept it.

We can fix `Copyright` in a different PR, if needed.